src/Client.php

```diff
@@ -247,5 +247,12 @@ private function _call($method, $params = array())
     {
         return $this->__soapCall($method, $params);
     }
+        public function getInvoiceMarkup($invoiceId, $type)
+    {
+       $obj = new AutotaskObjects\InvoiceMarkup;
```

Comment:
Needs parens before `;`, and lets use a more specific name than `$obj`.

`$invoiceMarkup = new AutotaskObjects\InvoiceMarkup();`

Also, I believe `AutotaskObjects\InvoiceMarkup` will need to be surfaced in the [class map found here|https://github.com/opendns/autotask-php/blob/master/src/Client.php#L7] (note alphabetical organization)